frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -56,15 +65,40 @@ export const BrowseModels = ({
 
   if (modelsFiltered.length) {
     return (
-      <GridContainer role="grid">
-        {groupsOfModels.map(groupOfModels => (
-          <ModelGroup
-            models={groupOfModels}
-            key={`modelgroup-${groupOfModels[0].collection.id}`}
-            localeCode={localeCode}
-          />
-        ))}
-      </GridContainer>
+      <>
+        {shouldShowBanner && (
+          <Banner role="complementary">
+            <BannerModelIcon>
+              <Icon name="model" />
+            </BannerModelIcon>
+            <Text size="md" lh="1rem" mr="1rem">
+              Models help curate data to make it easier to find answers to
```

Comment:
```suggestion
              Models curate data to make it easier to find answers to
```

Don't know if we need "help" in here seeing it now live. Makes it a little cleaner I think.

Also need to make sure this is translatable.

Reply:
@jeff-bruemmer Could you take a quick look at this sentence?